apps/cards.py

```diff
@@ -101,7 +101,31 @@ def card_lightcurve_summary():
                     )
                 )
             ),
-            help_popover(lc_help, 'help_lc'),
+            dmc.Group(
+                [
+                    dmc.Button(
+                        "Request data release photometry",
+                        id='lightcurve_request_release',
+                        variant="outline",
+                        color='gray',
+                        radius='xl', size='xs',
+                        compact=False,
+                    ),
+                    help_popover(
+                        lc_help,
```

Comment:
To make sure the users understand the new feature, the lightcurve helper should be expanded with:
- the reason for the new button (what it does)
- the description for the horizontal lines (`magnr`) in the `lc_help`

Reply:
Yes, sure! I just did not touch help text anywhere at all, waiting for your initial impression of the features / implementation. Will do.

Reply:
Added some documentation for these new things